src/functionaltests/java/com/ericsson/ei/notifications/trigger/SubscriptionNotificationSteps.java

```diff
@@ -74,11 +74,11 @@
     @Value("${spring.data.mongodb.database}")
     private String database;
 
-    @Value("${missedNotificationDataBaseName}")
-    private String missedNotificationDatabase;
+    @Value("${failedNotificationDataBaseName}")
+    private String failedNotificationDatabase;
 
-    @Value("${missedNotificationCollectionName}")
-    private String missedNotificationCollection;
+    @Value("${failedNotificationCollectionName}")
```

Comment:
If these two properties should be renamed then the documentation should reflect those updates and the example application.properties should be updated accordingly.